CONTRIBUTING.md

```diff
@@ -113,7 +113,7 @@ $ tar -xvzpf ensmallen-2.16.0.tar.gz
 $ cd ensmallen-latest
 ```
 
-Next, make a build directory and change into that directory.
+Next, make a build directory and change it into that directory.
```

Comment:
As above, I don't think "it" adds clarity here, and possibly makes it less clear.
```suggestion
Next, make a build directory and change into that directory.
```